# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/cubeb-coreaudio/todo.md
# Commit: eaa23cb3b396
# Full Hash: eaa23cb3b39686e114b1e78a78c2f8e5c91bcb29
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2019-12-28 04:06:02
# Regressor Bug: 1605471
# File Overlap Count: 5
# Description:
#   Bug 1605471 - Update cubeb-coreaudio to acb90e9. r=alwu
#   
#   Pick commits:
#   acb90e9 - Add license for mixer crate
#   82e7ff9 - Move criterion crate to [dev-dependencies]
# ==============================================================================

diff -r 509cd3b3ec22 -r eaa23cb3b396 third_party/rust/cubeb-coreaudio/todo.md
--- a/third_party/rust/cubeb-coreaudio/todo.md	Fri Dec 27 16:05:26 2019 +0000
+++ b/third_party/rust/cubeb-coreaudio/todo.md	Fri Dec 27 16:58:18 2019 +0000
@@ -71,6 +71,10 @@
   sub devices list of the aggregate device
 - Check the `name: CFStringRef` of the master device is not `NULL`
 
+### Mixer
+- Don't force output device to mono or stereo when the output device has one or two channel
+    - unless the output devicv is _Bose QC35, mark 1 and 2_.
+
 ## Interface to system types and APIs
 - Check if we need `AudioDeviceID` and `AudioObjectID` at the same time
 - Create wrapper for `AudioObjectGetPropertyData(Size)` with _qualifier_ info